test/python/test_jacobian.py

```diff
@@ -368,6 +368,43 @@ def test_net_rop_ddP(self):
         drop_num = self.rop_ddP(mode="net")
         self.assertNear(drop[self.rxn_idx], drop_num, self.rtol)
 
+    def rate_ddT(self, mode=None, const_p=False, rtol=1e-6):
+        # numerical derivative for production rates with respect to temperature
+        def calc():
+            if mode == "creation":
+                return self.gas.creation_rates
+            if mode == "destruction":
+                return self.gas.destruction_rates
+            if mode == "net":
+                return self.gas.net_production_rates
+
+        dt = self.tpx[0] * rtol
+        dp = 0 if const_p else self.tpx[1] * rtol
+        self.gas.TP = self.tpx[0] + dt, self.tpx[1] + dp
+        rate1 = calc()
+        self.gas.TP = self.tpx[:2]
+        rate0 = calc()
+        return (rate1 - rate0) / dt
+
+    def test_net_rate_ddT(self):
+        # check derivatives of net creation rates with respect to temperature
```

Comment:
I think it would help to explicitly write the identity that's being checked here.